charts/warpstream-agent/values.yaml

```diff
@@ -21,6 +25,13 @@ deploymentStrategy:
     # The number of pods that can be unavailable during the update process (both integers and percentages are supported)
     maxUnavailable: 1
 
+statefulSetConfig:
+  clusterDomain: cluster.local
+  podManagementPolicy: OrderedReady  # or Parallel, see https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#pod-management-policies
```

Comment:
Default Parallel? It says it only impacts scaling not updates, so we want Parallel as it will make scaling up faster and have basically no other downside

Reply:
Oh yep, can do that.